sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -298,7 +298,7 @@ object RapidsConf {
 
   // Resource Configuration
 
-  val PINNED_POOL_SIZE = conf("spark.rapids.memory.pinnedPool.size")
+  val PINNED_POOL_SIZE = conf("spark.rapids.memory.pinnedPool.size-DUMMY_MODIFIED")
```

Comment:
this one failed (change to RpiadConf only but not config.md) as expected [build 1023](https://blossom.nvidia.com/sw-gpu-spark-jenkins/job/rapids_premerge-github/1023/console)
![image](https://user-images.githubusercontent.com/8086184/109110729-a6011d00-7772-11eb-81df-d2e1b8a4c85c.png)